service/r2/service.go

```diff
@@ -75,6 +77,7 @@ type serviceMetrics struct {
 	updateRollupRule        instrument.MethodMetrics
 	deleteRollupRule        instrument.MethodMetrics
 	fetchRollupRuleHistory  instrument.MethodMetrics
+	bulkUpdateRuleSet       instrument.MethodMetrics
```

Comment:
Perhaps we just call this `updateRuleSet` since you removed `bulk` from the swagger so we can be consistent to not include bulk in the method name?

Reply:
Ping